app/assets/javascripts/application.js

```diff
@@ -23,8 +23,14 @@ APP = {
 			// $(document).foundationTopBar();
       $("a.check-in")
         .bind("ajax:beforeSend", function(e, xhr) {
-          $(e.target).parent("span.check_in")
-            .html("<i class=\"fi-check\"></i> Checked In");
+          $target = $(e.target).parent("span.check_in");
+          event_class = $target.attr("class").split(" ")[1];
```

Comment:
Identifier 'event_class' is not in camel case.<br>'event_class' is not defined.<br>'$target' is not defined.